plugins/@grouparoo/app-templates/src/source/table/pluginMethods.ts

```diff
@@ -24,8 +30,6 @@ export enum FilterOperation {
   NotContain = "does not contain",
 }
 
-export type DataResponse = string | number | boolean | Date;
-
 export interface MatchCondition {
   columnName: string;
   filterOperation: FilterOperation;
```

Comment:
The `relative*` ProfilePropertyRuleFiltersWithKey keys are for relative dates, ie: "greater than 3 days ago".  This logic is implemented in Group rules, but not yet in filters.

Reply:
Well cool. Then we can change it here in one place if it does!